### ISSUE TITLE
Push images to ECR

### ISSUE THREAD

**Issue by sriram-lmnd** [softwaremill/elasticmq#938]
Hello!

AWS ECR contains several [old/unofficial versions of `elasticmq` and `elasticmq-native`](https://gallery.ecr.aws/search?searchTerm=elasticmq). Are there any plans to officially push images to ECR?

**Comment by micossow** [comment]
No, we do not plan to do that. Are you not able to access the images on Docker Hub?

**Comment by sriram-lmnd** [comment]
I'm running into rate limits when trying to incorporate images from Docker Hub into my build process. I was able to find all our other dependencies on ECR's public gallery, but not Elasticmq. Thus the question.
